<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recognizer</class>
 <widget class="QMainWindow" name="Recognizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recognizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/Main/searching33.png</normaloff>:/Main/searching33.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="SettingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QGroupBox" name="Update_groupBox">
          <property name="title">
           <string>Database Update (For LBPH Only)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0,0,0,0">
             <item>
              <widget class="QCheckBox" name="UpdateExisting_checkBox">
               <property name="text">
                <string>Adding more pictures to existing</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="UpdateExistingLabel">
               <property name="text">
                <string>Known label:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="UpdateExisting_spinBox"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_12"/>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1,8,1">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="UpdateLocation_label">
               <property name="text">
                <string>Location:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="UpdateLocation_lineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="UpdateLocation_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="TrainNew_pushButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Update Trainning</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="Prediction_groupBox">
          <property name="title">
           <string>Prediction</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="Target_label">
               <property name="text">
                <string>Target</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Target_lineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="Target_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="Predition_Result">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Predict_Button">
               <property name="text">
                <string>Predict</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Camera Settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="CameraNumberLabel">
               <property name="text">
                <string>Camera #</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="CameraNumber"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="CameraUpdateLabel">
               <property name="text">
                <string>Frame Updating rate (ms/updating)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="CameraUpdateTime">
               <property name="maximum">
                <number>3000</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="ClassifierDurationlabel">
               <property name="text">
                <string>Classifier duration (ms)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="ClassifierDuration">
               <property name="maximum">
                <number>5000</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="Preparation_groupBox">
          <property name="title">
           <string>Preparation</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="CVS_Path_label">
                 <property name="text">
                  <string>CVS Path</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="CVS_Path_lineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QToolButton" name="CVS_Path_Button">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="CreateCVS_Button">
               <property name="text">
                <string>Create CVS</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="ClassifierPathLabel">
               <property name="text">
                <string>Classifier Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ClassifierLineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="ClassifierPath_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBox" name="LBPH_checkBox">
               <property name="text">
                <string>LBPH</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LBPH_lineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="LBPH_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QCheckBox" name="EigenFace_checkBox">
               <property name="text">
                <string>EigenFace</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="EigenFace_location_lineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="EigenFace_location_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QCheckBox" name="FisherFace_checkBox">
               <property name="text">
                <string>FisherFace</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="FisherFace_location_lineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="FisherFace_location_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="SaveTrainedModel_pushButton">
               <property name="text">
                <string>Save Trained Model</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LoadTrainedModel_pushButton">
               <property name="text">
                <string>Load Trained Model</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="Train_Button">
               <property name="text">
                <string>Train</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
